src/pages/settings/schedules/create/Create.tsx

```diff
@@ -94,7 +99,9 @@ export function Create() {
       setErrors(undefined);
       toast.processing();
 
-      request('POST', endpoint('/api/v1/task_schedulers'), schedule)
+      const formattedSchedule = formatSchedulePayment(schedule!);
```

Comment:
Is it possible to if-check the schedule instead of forcing it to be a valid value? Or perhaps return early. We wanna make sure value is there.

Reply:
Sure, added checking condition just next to !isFormBusy.